components/camel-pulsar/src/main/java/org.apache.camel.component.pulsar/configuration/AdminConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.camel.component.pulsar.configuration;
+
+import org.apache.pulsar.client.impl.conf.ClientConfigurationData;
+
+import java.util.Set;
+
+public class AdminConfiguration extends ClientConfigurationData {
```

Comment:
Might be best to extract all admin related configuration such as tenant and namespace into the AdminConfiguration file?

Reply:
The ClientConfigurationData is what Pulsar expects to be passed in to configure security etc. for the PulsarAdmin API. I could make ClientConfigurationData a field of AdminConfiguration. Waht do you think @AbuahK ?

Reply:
Tenant and namespace are specified by the route and so can't be in the AdminConfiguration.